pg-extend/src/pg_fdw.rs

```diff
@@ -8,7 +8,7 @@
 
 // FDW on PostgreSQL 11+ is not supported. :(
 // If anyone tries to enable "fdw" feature with newer Postgres, throw error.
-#[cfg(feature = "postgres-11")]
+#[cfg(any(feature = "postgres-11", feture = "postgres-12"))]
```

Comment:
would you mind terribly if i asked whether you really meant `feature` here?

Reply:
Thanks for spotting that :)